www/src/pages/guidelines/color.js

```diff
@@ -0,0 +1,270 @@
+  ContentColumn,
+} from "../../components/guidelines/containers"
+import {
+  Intro,
+  PageHeading,
+  SectionHeading,
+  SectionSubheading,
+} from "../../components/guidelines/typography"
+
+import Badge from "../../components/guidelines/badge"
+import Overview from "../../components/guidelines/color/overview"
+import ColorModal from "../../components/guidelines/color/modal"
+
+import palette from "../../utils/guidelines/extend-palette-info"
+
+// http://reactcommunity.org/react-modal/accessibility/
+Modal.setAppElement(`#___gatsby`)
+
+const LegacyColorIcon = ({ textColor }) => (
+  <Box
+    borderRadius={7}
+    fontSize={0}
+    lineHeight="solid"
+    height="8px"
+    width="8px"
+    display="inline-block"
+    bg={textColor}
+  />
+)
+class Color extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isModalOpen: false,
+      color: false,
+    }
+  }
+
+  handleModalOpen = (event, node) => {
+    event.persist()
+    document.querySelector(`html`).style.overflowY = `hidden`
+    this.setState({ isModalOpen: true, color: node })
+  }
+
+  handleModalClose = event => {
+    event.persist()
+    document.querySelector(`html`).style.overflowY = `auto`
+    this.setState({ isModalOpen: false })
+  }
+
+  render() {
+    return (
+      <Layout pathname={this.props.location.pathname} pageTitle="Color">
+        <Container>
+          <PageHeading>Color</PageHeading>
+          <Intro>
+            Vibrant, friendly, and accessible color is an important part of
+            Gatsby’s design. It distinguishes our brand and helps us to create
+            consistent experiences and meaningful expressions across marketing
+            and products.
+          </Intro>
+          <Badge my={3}>
+            Work in Progress{` `}
+            <MdWarning style={{ fontSize: 16, marginLeft: `0.25rem` }} />
+          </Badge>
+        </Container>
+
+        <Section>
+          <Columns>
+            <CopyColumn sticky={false}>
+              <Text as="p" mb={0}>
+                Our color palette includes primary and secondary colors that can
+                be used for interfaces as well as illustrations.
+                <br />
+                <strong>Click the color names for detailed information.</strong>
+              </Text>
+            </CopyColumn>
+            <ContentColumn alignSelf="flex-end">
+              <Flex alignItems="flex-end">
+                <Flex alignItems="center" mr={8}>
+                  <LegacyColorIcon textColor="grey.50" />
+                  {` `}
+                  <Text as="span" ml={2} c>
+                    Aliased color
+                  </Text>
+                </Flex>
+
+                <Box>
+                  <Text
+                    as="h4"
+                    color="grey.50"
+                    mb={0}
+                    mt={{ xxs: 8, lg: 0 }}
+                    fontWeight={0}
+                  >
+                    WCAG Contrast Score
```

Comment:
I love the contrast score! Can this heading be an h3 to keep the hierarchy going in order?

Reply:
👌 c21642b